docs/resources/Channel.md

```diff
@@ -6,26 +6,27 @@ Represents a guild or DM channel within Discord.
+| bitrate?               | integer                                                                                                      | the bitrate (in bits) of the voice channel                                                                                                                                      |
+| user_limit?            | integer                                                                                                      | the user limit of the voice channel                                                                                                                                             |
+| rate_limit_per_user?   | integer                                                                                                      | amount of seconds a user has to wait before sending another message (0-21600); bots, as well as users with the permission `manage_messages` or `manage_channel`, are unaffected |
+| recipients?            | array of [user](#DOCS_RESOURCES_USER/user-object) objects                                                    | the recipients of the DM                                                                                                                                                        |
+| icon?                  | ?string                                                                                                      | icon hash                                                                                                                                                                       |
+| owner_id?              | snowflake                                                                                                    | id of the DM creator                                                                                                                                                            |
+| application_id?        | snowflake                                                                                                    | application id of the group DM creator if it is bot-created                                                                                                                     |
+| parent_id?             | ?snowflake                                                                                                   | id of the parent category for a channel (each parent category can contain up to 50 channels)                                                                                    |
+| last_pin_timestamp?    | [ISO8601 timestamp](https://www.loc.gov/standards/datetime/iso-tc154-wg5_n0038_iso_wd_8601-1_2016-02-16.pdf) | when the last pinned message was pinned                                                                                                                                         |
```

Comment:
rather than linking to this document in multiple places, can we add a definition in our own docs which we give a brief explanation of and then link to this document?

Reply:
Hm, where would a comment like this fit? Maybe I should put this comment under one of the models that use an ISO8601 timestamp, but which one... Any ideas?

Reply:
you can place it on the reference page, which describes other generalities of the apis

Reply:
Alright, I think I did this pretty well (put it under the "HTTP API" section in the reference)